src/main/kotlin/no/nav/familie/tilbake/oppgave/OppgaveService.kt

```diff
@@ -152,7 +153,7 @@ class OppgaveService(
         val oppgaveResponse = integrasjonerClient.opprettOppgave(opprettOppgave)
         antallOppgaveTyper[oppgavetype]!!.increment()
         log.medContext(logContext) {
-            info("Ny oppgave (id=${oppgaveResponse.oppgaveId}, type=$oppgavetype, frist=$fristForFerdigstillelse) opprettet for behandling $behandlingId")
+            info("Ny oppgave (id=${oppgaveResponse.oppgaveId}, type=$oppgavetype, frist=$fristForFerdigstillelse) opprettet for behandling ${behandling.id}")
```

Comment:
```suggestion
            info("Ny oppgave (id={}, type={}, frist={}) opprettet for behandling {}", oppgaveResponse.oppgaveId, oppgavetype, fristForFerdigstillelse, behandling.id)
```

Mente @sillerud viste meg dette tidligere denne uken. Gjelder for så vidt for alle tilfeller. Stemmer det?

Reply:
Jepp, man burde _egentlig_ ikke bruke kotlin string templating for loggmeldinger. Det går fint i logback, men er IMO et anti pattern